go/consensus/tendermint/apps/staking/state/state.go

```diff
@@ -424,8 +428,8 @@ func (s *ImmutableState) Slashing(ctx context.Context) (map[staking.SlashReason]
 	return params.Slashing, nil
 }
 
-func (s *ImmutableState) LastBlockFees(ctx context.Context) (*quantity.Quantity, error) {
-	value, err := s.is.Get(ctx, lastBlockFeesKeyFmt.Encode())
+func (s *ImmutableState) loadStoredBalance(ctx context.Context, key *keyformat.KeyFormat) (*quantity.Quantity, error) {
```

Comment:
Should this also be used for `CommonPool` and `TotalSupply`?

Reply:
good idea, will update